src/Observable.php

```diff
@@ -87,9 +117,27 @@ public function subscribe(ObserverInterface $observer, $scheduler = null)
     }
 
     /**
+     * @deprecated
+     *
+     * @param callable|null $onNext
+     * @param callable|null $onError
+     * @param callable|null $onCompleted
+     * @return DisposableInterface
+     */
+    public function subscribeCallback(callable $onNext = null, callable $onError = null, callable $onCompleted = null): DisposableInterface
+    {
+        $observer = new CallbackObserver($onNext, $onError, $onCompleted);
+
+        return $this->subscribe($observer);
+    }
+
+    /**
      * @internal
+     *
+     * @param ObserverInterface $observer
+     * @return bool
      */
-    public function removeObserver(ObserverInterface $observer)
+    public function removeObserver(ObserverInterface $observer): bool
```

Comment:
It probably only belongs on Subjects anyway, since an observable will only ever have one observer